fdb-relational-api/src/testFixtures/java/com/apple/foundationdb/relational/utils/OptionsTestHelper.java

```diff
@@ -0,0 +1,75 @@
+
+/**
+ * Test fixtures related to {@link Options}.
+ */
+public class OptionsTestHelper {
+
+    public static Options nonDefaultOptions() throws SQLException {
+        Options.Builder builder = Options.builder();
+        // Cannot do CONTINUATION, since there is no implementation available.
+        builder = builder.withOption(Options.Name.INDEX_HINT, "thisIndex");
+        builder = builder.withOption(Options.Name.MAX_ROWS, 5);
+        builder = builder.withOption(Options.Name.REQUIRED_METADATA_TABLE_VERSION, 123);
+        builder = builder.withOption(Options.Name.TRANSACTION_TIMEOUT, 5000L);
+        builder = builder.withOption(Options.Name.INDEX_FETCH_METHOD, Options.IndexFetchMethod.USE_REMOTE_FETCH);
+        builder = builder.withOption(Options.Name.DISABLE_PLANNER_REWRITING, true);
+        builder = builder.withOption(Options.Name.DISABLED_PLANNER_RULES, List.of("r1", "r2"));
+        builder = builder.withOption(Options.Name.PLAN_CACHE_PRIMARY_MAX_ENTRIES, 512);
+        builder = builder.withOption(Options.Name.PLAN_CACHE_PRIMARY_TIME_TO_LIVE_MILLIS, 1000L);
+        builder = builder.withOption(Options.Name.PLAN_CACHE_SECONDARY_MAX_ENTRIES, 128);
+        builder = builder.withOption(Options.Name.PLAN_CACHE_SECONDARY_TIME_TO_LIVE_MILLIS, 3000L);
+        builder = builder.withOption(Options.Name.PLAN_CACHE_TERTIARY_MAX_ENTRIES, 4);
+        builder = builder.withOption(Options.Name.PLAN_CACHE_TERTIARY_TIME_TO_LIVE_MILLIS, 2000L);
+        builder = builder.withOption(Options.Name.REPLACE_ON_DUPLICATE_PK, true);
+        builder = builder.withOption(Options.Name.LOG_QUERY, true);
+        builder = builder.withOption(Options.Name.LOG_SLOW_QUERY_THRESHOLD_MICROS, 1000000L);
+        builder = builder.withOption(Options.Name.EXECUTION_SCANNED_BYTES_LIMIT, 500000L);
+        builder = builder.withOption(Options.Name.EXECUTION_TIME_LIMIT, 100L);
+        builder = builder.withOption(Options.Name.EXECUTION_SCANNED_ROWS_LIMIT, 4000);
+        builder = builder.withOption(Options.Name.DRY_RUN, true);
+        builder = builder.withOption(Options.Name.CASE_SENSITIVE_IDENTIFIERS, true);
+        builder = builder.withOption(Options.Name.CURRENT_PLAN_HASH_MODE, "m1");
+        builder = builder.withOption(Options.Name.VALID_PLAN_HASH_MODES, "m1,m2");
+        builder = builder.withOption(Options.Name.CONTINUATIONS_CONTAIN_COMPILED_STATEMENTS, false);
+        builder = builder.withOption(Options.Name.ASYNC_OPERATIONS_TIMEOUT_MILLIS, 5000L);
+        builder = builder.withOption(Options.Name.ENCRYPT_WHEN_SERIALIZING, true);
+        builder = builder.withOption(Options.Name.ENCRYPTION_KEY_STORE, "secrets.ks");
+        builder = builder.withOption(Options.Name.ENCRYPTION_KEY_ENTRY, "mykey");
+        builder = builder.withOption(Options.Name.ENCRYPTION_KEY_PASSWORD, "mypass");
+        Options options = builder.build();
+        for (Options.Name name : Options.Name.values()) {
+            if (name != Options.Name.CONTINUATION) {
```

Comment:
Maybe add a comment that continuation is not supported for string conversion?

Reply:
There was a comment at the front of the method, but I added another one here and made those comments more explicit about the module dependency issue.